### PR TITLE
Add Caroso cali recommendations

### LINKED ISSUE #21: Add caroso Cali file
- [ ] Create a **branch**
- [ ] _Add file/commits_
- [ ] _Open a PR_
- [ ] Have a discussion about the PR (COLLABORATE)
- [ ] Make edits
- [ ] Get :shipit: approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by caroso**
closes #21 
hey, @WilburHughes , does this look like a good list of recommendations

### CODE REVIEW

**Comment by WilburHughes** [comment]
this look great. check it in. @caroso

File: caroso-cali.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Cali
+###Place to eat
+-Unicentro
```
Comment (WilburHughes) [inline caroso-cali.md:3]: this is a test of places to go :+1:  **bold text here**

Comment (WilburHughes) [reply caroso-cali.md:3]: this is a added line note :8ball:

**Comment by caroso** [comment]
Hi @WilburHughes , I added a new place to avoid

**Comment by WilburHughes** [comment]
:+1:

**Pull request merged**